index.bs

```diff
@@ -7212,486 +7203,6 @@ However, [=authenticators=] that do not utilize [[!FIDO-CTAP]] do not necessaril
 : Authenticator extension processing
 :: [=largeblob|This extension=] directs the user-agent to cause the large blob to be stored on, or retrieved from, the authenticator. It thus does not specify any direct authenticator interaction for [=[RPS]=].
 
-
-## Authenticator Extensions ## {#sctn-defined-authenticator-extensions}
-
-This section defines extensions that are both [=client extensions=] and [=authenticator extensions=].
```

Comment:
Hm, I think we should probably keep this as a placeholder (adding a note of this)? That makes it clear that the section is intentionally empty rather than accidentally dropped somehow, and also keeps the section anchor working in this document version.

```
## Authenticator Extensions ## {#sctn-defined-authenticator-extensions}

This section defines extensions that are both [=client extensions=] and [=authenticator extensions=].

This section is currently empty.
```